lib/CL/pocl_binary.c

```diff
@@ -818,9 +897,48 @@ pocl_binary_get_kernels_metadata (cl_program program, unsigned device_i)
       km->has_arg_metadata = k.has_arg_metadata;
       km->name = k.kernel_name;
       km->data
-          = (void **)calloc (program->associated_num_devices, sizeof (void *));
+          = (void **)calloc (program->associated_num_devices, sizeof(void *));
       assert (km->name);
 
+      if (k.flags & POCL_KERNEL_HAS_SUBG_META) {
```

Comment:
Clang-format? Doesn't look like GNU C style.

Reply:
yep these were some of the last fixes, i'll rerun the clang-format.

Reply:
fixed